samples/connectedhomeip/light_bulb/src/LightBulbPublishService.cpp

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+#include "LightBulbPublishService.h"
+
+#include "AppTask.h"
+
+#include <logging/log.h>
+#include <zephyr.h>
+
+#include <openthread/message.h>
+#include <openthread/udp.h>
+#include <platform/OpenThread/OpenThreadUtils.h>
+
+#include <inet/InetConfig.h>
+#include <platform/CHIPDeviceLayer.h>
+#include <transport/TransportMgr.h>
+#include <transport/raw/UDP.h>
+
+LOG_MODULE_DECLARE(app);
+
+// Pointer to the LightBulbPublishService instance needed by static
+// PublishTimerHandler method
+static LightBulbPublishService* sLightBulbPublishService;
+
+const char PUBLISH_SERVICE_PAYLOAD[] = "chip-light-bulb";
```

Comment:
I kind of assumed that we prefer using kCamelCase for constants and changed the lock sample that way. We probably can unify the style when we prepare the final PR.

Reply:
Ok and previously I tried to align to your style :) So let's use it kCamelCase then.